CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -18,7 +18,8 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 - Fix [CVE-2021-28235](https://nvd.nist.gov/vuln/detail/CVE-2021-28235) by [clearing password after authenticating the user](https://github.com/etcd-io/etcd/pull/15653).
 
 ### Package `netutil`
-- Fix [consistently format IPv6 addresses for comparison](https://github.com/etcd-io/etcd/pull/15187)
+- Fix [consistently format IPv6 addresses for comparison](https://github.com/etcd-io/etcd/pull/15187).
+- Fix [etcdserver may panic when parsing a JWT token without username or revision](https://github.com/etcd-io/etcd/pull/15676).
```

Comment:
Should this be under `netutil` or `etcdserver`?  Thinking it should align with 3.4, apologies if I am misunderstanding something.

Reply:
Good catch, updated.

I meant to put it under etcdserver, somehow put the wrong place.